src/main/java/lotto/domain/Lottos.java

```diff
@@ -1,36 +1,35 @@
 package lotto.domain;
 
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
-import java.util.stream.IntStream;
+
+import static lotto.util.NullValidator.validateNull;
 
 public class Lottos {
-    private final List<Lotto> lottos;
+    private final List<Lotto> elements;
 
-    public Lottos(LottoMoney money) {
-        validateNull(money);
-        int lottoCount = money.calculateLottoCount();
-        this.lottos = IntStream.range(0, lottoCount)
-                .mapToObj(index -> Lotto.create())
-                .collect(Collectors.toList());
+    public Lottos(List<Lotto> elements) {
+        validateNull(elements);
+        this.elements = elements;
     }
 
-    private static void validateNull(LottoMoney money) {
-        if (money == null) {
-            throw new RuntimeException("null이 입력되었습니다.");
-        }
+    public static Lottos combine(Lottos lottos1, Lottos lottos2) {
```

Comment:
불용어는 안쓰는게 좋아요
새로운 로또 생성 규칙이 생기면 lottos3이라 썼을 때 무슨 규칙인지 바로 파악이 안돼서 코드를 분석해봐야해요

```suggestion
    public static Lottos combine(List<Lottos> lottos) {
```